cfme/common/vm_views.py

```diff
@@ -545,15 +546,52 @@ class PolicySimulationView(BaseLoggedInPage):
     """
     Policy Simulation page for vms/instances
     """
+
+    title = Text("#explorer_title_text")
+
     @View.nested
     class form(View):  # noqa
-        policy = BootstrapSelect('policy_id')
+        policy_profile = BootstrapSelect("profile_id")
         # TODO policies table, ability to remove
         entities = View.nested(BaseNonInteractiveEntitiesView)
-        cancel_button = Button('Cancel')
+        cancel_button = Button("Cancel")
 
-    # TODO match quadicon
-    is_displayed = displayed_not_implemented
+    @property
+    def quadicon(self):
```

Comment:
Required:
I would not use `quadicon`
1. it's part of `PolicySimulationView` means, I can select multiple `vms` try simulation form `All VM page`. 
2. we have entities so I would use `view.entities` for all operations to increase readability and consistency.

Reply:
@digitronik Since `PolicySimulationView` is registered against an instance/vm, I think it is important to check the quadicon, since it is the only widget on the page that is specific to the instance/vm. 

If we have a policy simulation against a number of items, I think this should be a different view that is registered against the collection rather than the vm/instance and therefore has a different `is_displayed` method.

Reply:
Myview:
@john-dupuy we register navigation destination against any `BaseEntity` not view.
Entity using `BaseNonInteractiveEntitiesView` means it capable to provide you `n` numbers of `vm/Instance` on that page. with such implementation we are restricting for single `vm/Instance`.

Except entity not changes in view as per me. Let me know why we need other view than this?
You can check entity name in `test` for more validation. so in future, we can parametrize this test in better way. even you are parametrizing this test with `all` and `detail` page.